doc/manual/command-ref/conf-file.xml

```diff
@@ -864,6 +864,26 @@ requiredSystemFeatures = [ "kvm" ];
 
   </varlistentry>
 
+  <varlistentry xml:id="conf-tarball-ttl"><term><literal>tarball-ttl</literal></term>
+
+    <listitem>
+
+      <para>Default: <literal>3600</literal> seconds.</para>
+
+      <para>The number of seconds a download is cached in
+      <filename>$XDG_CACHE_HOME/nix/tarballs</filename>
+      are considered fresh.
```

Comment:
`is cached in ... are considered fresh` -> `... is considered fresh`. And maybe add something like "If a cached tarball is older, Nix will check whether it is still up to date using its ETag header, and if not, download the latest version."